x-pack/plugins/integration_assistant/public/components/create_integration/create_integration_assistant/steps/data_stream_step/use_generation.tsx

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isEmpty } from 'lodash/fp';
+import { useCallback, useEffect, useState } from 'react';
+import type { HttpSetup } from '@kbn/core-http-browser';
+import { getLangSmithOptions } from '../../../../../common/lib/lang_smith';
+import type { Docs, ESProcessorItem, Pipeline, SamplesFormat } from '../../../../../../common';
+import {
+  type AnalyzeLogsRequestBody,
+  type CategorizationRequestBody,
+  type EcsMappingRequestBody,
+  type RelatedRequestBody,
+} from '../../../../../../common';
+import {
+  runCategorizationGraph,
+  runEcsGraph,
+  runRelatedGraph,
+  runAnalyzeLogsGraph,
+} from '../../../../../common/lib/api';
+import { useKibana } from '../../../../../common/hooks/use_kibana';
+import type { State } from '../../state';
+import * as i18n from './translations';
+import { useTelemetry } from '../../../telemetry';
+import type { ErrorCode } from '../../../../../../common/constants';
+import type { AIConnector, IntegrationSettings } from '../../types';
+
+export type OnComplete = (result: State['result']) => void;
+export const ProgressOrder = ['ecs', 'categorization', 'related'];
+export type ProgressItem = (typeof ProgressOrder)[number];
```

Comment:
```suggestion
const ProgressOrder = ['analyzeLogs', 'ecs', 'categorization', 'related'] as const;
type ProgressItem = typeof ProgressOrder[number];
```

Reply:
Fixed in https://github.com/elastic/kibana/pull/194150/commits/93eb9d54bff48530f018b7dd919273177b844879